### PR TITLE
set CODEOWNERS on admin-ui-components

### PR DESCRIPTION
**Pull request by dhartunian**
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/ui/168)

### CODE REVIEW

**Review by nathanstilwell — APPROVED** [review]
LGTM

**Review by nathanstilwell — CHANGES_REQUESTED** [review]
Actually, I take it back. If I'm reading [this example](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners) right, the last match wins so if you put the `admin-ui-components` package after the global owners, then only the admin owners will be

File: CODEOWNERS [hunk 1]
```diff
@@ -14,4 +14,5 @@
 # In each subsection folders are ordered first by depth, then alphabetically.
 # This should make it easy to add new rules without breaking existing ones.
 
+packages/admin-ui-components @dhartunian @nkodali @nathanstilwell
 *   @nathanstilwell @j-low @laurenbarker
```
Comment (nathanstilwell) [inline CODEOWNERS:18]:
```suggestion
*   @nathanstilwell @j-low @laurenbarker
packages/admin-ui-components @dhartunian @nkodali @nathanstilwell
```

Comment (dhartunian) [reply CODEOWNERS:18]: ah thanks for actually reading the docs!

**Pull request merged**